multi_drone_sar/DroneInfo[] info
